Move binaries to /bin instead of /sbin
It can be executed by normal users without privilegesIndex: lshw-02.14/src/Makefile
===================================================================
--- a/src/Makefile
+++ b/src/Makefile
@@ -6,7 +6,7 @@
 
 DESTDIR?=/
 PREFIX?=/usr
-SBINDIR=$(PREFIX)/sbin
+SBINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
 DATADIR=$(PREFIX)/share
 INSTALL?=install -p
